src/main/java/com/sap/sgs/phosphor/fosstars/data/github/UsesGithubForDevelopment.java

```diff
@@ -38,17 +39,18 @@
         return StringUtils.isEmpty(description) || !description.toLowerCase().contains("mirror");
       },
 
+      // check if the repository doesn't have an explicit link to the original repository
+      // or an explicit link to an SVN repository and the link does not contain github.com
+      repository -> Stream.of(repository.getMirrorUrl(), repository.getSvnUrl())
+          .filter(StringUtils::isNotEmpty)
+          .noneMatch(UsesGithubForDevelopment::notGitHubUrl),
```

Comment:
Please correct me if I am wrong, .noneMatch(UsesGithubForDevelopment::notGitHubUrl) would return true if all of the urls are github right? Because, noneMatch() returns true if none of the stream elements match the given predicate, it returns false if any of the stream elements matches the condition specified by the predicate.

Reply:
And if yes, this conflicts the above comment 
// or an explicit link to an SVN repository and the link does not contain github.com
right?

Reply:
The check works as you described. I don't think that the comment conflicts with the check but I find the comment quite quite difficult to read. Therefore, I updated the comment. Hope it's clearer now.